Modules/Microsoft365DSC/DSCResources/MSFT_PlannerTask/MSFT_PlannerTask.psm1

```diff
@@ -784,8 +784,34 @@ function Export-TargetResource
                         $currentDSCBlock += '        PlannerTask ' + (New-Guid).ToString() + "`r`n"
                         $currentDSCBlock += "        {`r`n"
                         $content = Get-DSCBlock -Params $result -ModulePath $PSScriptRoot
-                        $content = Convert-DSCStringParamToVariable -DSCBlock $content `
-                            -ParameterName 'Credential'
+
```

Comment:
Please have a look at these two functions. They are used in other resources and do combine the steps below:

$Results = Update-M365DSCExportAuthenticationResults -ConnectionMode $ConnectionMode `
                    -Results $Results
                $currentDSCBlock = Get-M365DSCExportContentForResource -ResourceName $ResourceName `
                    -ConnectionMode $ConnectionMode `
                    -ModulePath $PSScriptRoot `
                    -Results $Results `
                    -Credential $Credential

Reply:
@andikrueger Fixed in latest commit.